Lombiq.NodeJs.Extensions.Samples/Lombiq.NodeJs.Extensions.Samples.csproj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
```

Comment:
This should rather be 8.0: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults.

Reply:
But why? `netstandard2.0` is supported up until C# 7.3 according to the docs. C# 8 is from netstandard2.1 which will produce unsupported language feature warnings most probably.

Reply:
Ah, it's 2.**1**, indeed.